fud/fud/stages/vivado/extract.py

```diff
@@ -69,6 +70,27 @@ def futil_extract(directory):
         f8_muxes = to_int(find_row(slice_logic, "Site Type", "F8 Muxes")["Used"])
         f9_muxes = to_int(find_row(slice_logic, "Site Type", "F9 Muxes")["Used"])
 
+        resourceInfo.update({
```

Comment:
Cool! It would be nice to add a few comments here explaining what these two different `update` calls are doing.

Reply:
I'd say that it's close to self-explanatory, but i'll add some comments.

Reply:
I disagree. What is this information in each update call pertinent to? Why is this necessary? 

How come you split these two stages? What's going to stop someone from seeing these two `update` calls and think, "huh, these can just be combined. Let me refactor this real quick."

Reply:
Just to clarify, your change didn't add the majority of the code, so its not strictly required. At the very least, we should document why these are split however.